### ISSUE TITLE
AUDIT - Settings Zoom

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyterlab-accessible-themes#16]
- [ ] 200%
- [ ] 400%
- [ ] Change the default value of text size from 16 to 30 in the web browser
- [ ] Test with different screen sizes

## References

- [Theme scope checklist](https://github.com/isabela-pf/accessibility/pull/1/files#diff-084cbb6818caeb52700399405bb8c8e8ed366445ad27c7ac83c70aa10f519382)
- [Audit spreadsheet](https://docs.google.com/spreadsheets/d/1yaWY--2zBi7KqPAnMMbJp_aVXFFT8sBDdci8UfpKjO4/edit#gid=0)

**Comment by steff456** [comment]
These are the results of the Zoom Audit,

## Settings behavior

<img width="1317" alt="image" src="https://user-images.githubusercontent.com/20992645/183122670-66ecfb85-7226-4a48-8264-892c287d0ba1.png">

## Notes

- Settings is not usable with 200% or 400% zoom, the options are chopped and not readable
- The area in the container is defined in px and it changes dynamically with the screen size
- Changing the default text size doesn't do any effect in Jlab UI, but changing the minimum text size in the browser does create changes.
- The width of the left menu area is fixed in px - this is the reason why it doesn't resize
- Modifying to responsive layouts break the settings view and its usability
- The interface is using the same gear icon for multiple entries
- In the case that the user expands all the sidebars in small screens or with high zoom, the area of the settings and main container is reduced to nothing making it not usable.

## Suggestions
- Adding a way to compress the left menu area or adding a horizontal scrollbar so the user will be able to navigate through all the options
- Do we want to have the same icon for multiple entries in the setting's menu?
- We need to define a new UX for when the left/right sidebar is expanded
